marimo/_save/stores/redis.py

```diff
@@ -22,13 +15,14 @@ def __init__(self, **kwargs: Any) -> None:
         # See list of options here:
         self.redis = redis.Redis(**kwargs)
 
-    def get(self, key: HashKey, loader: Loader) -> Optional[bytes]:
-        return self.redis.get(str(loader.build_path(key))) or None
+    def get(self, key: str) -> Optional[bytes]:
+        result = self.redis.get(key)
+        if result is None:
+            return None
+        return result.encode("utf-8")  # type: ignore[no-any-return]
```

Comment:
is encode right here? Not sure what typing is suggesting, I thought `.get` returned bytes already

Reply:
get returns a string. i wasn't sure what the right encoding was. do you know?